lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -154,6 +154,8 @@ describe('ContentMetadataComponent', () => {
 
     const queryDom = (properties = 'properties') => fixture.debugElement.query(By.css(`[data-automation-id="adf-metadata-group-${properties}"]`));
 
+    const emptyPanelMessage = () => fixture.debugElement.query(By.css('.adf-metadata-no-item-added'));
```

Comment:
instead of referring directly to debugElement please reuse UnitTestingUtils which Michał created (you can find multiple examples of that in tests)